python/matplotlib.ipynb

```diff
@@ -1,748 +1,730 @@
+                "Manchmal braucht man einen Polarplot:"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "#r = np.linspace(0, 10, 1000)\n",
+                "r = np.linspace(0, 10, 50)\n",
+                "theta = 2 * np.pi * r\n",
+                "\n",
+                "plt.polar(theta, r)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "# Ticks\n",
+                "Man kann sehr viele Sachen mit Ticks machen…"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "x = np.linspace(0, 2 * np.pi)\n",
+                "\n",
+                "plt.plot(x, np.sin(x))\n",
+                "plt.xlim(0, 2 * np.pi)\n",
+                "# erste Liste: Tick-Positionen, zweite Liste: Tick-Beschriftung\n",
+                "plt.xticks([0, np.pi / 2, np.pi, 3 * np.pi / 2, 2 * np.pi],\n",
+                "           [r\"$0$\", r\"$\\frac{1}{4}\\tau$\", r\"$\\frac{1}{2}\\tau$\", r\"$\\frac{3}{4}\\tau$\", r\"$\\tau$\"])\n",
+                "plt.title(r\"$\\tau$ FTW!\")"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {
+                "collapsed": true
+            },
+            "outputs": [],
+            "source": [
+                "months = ['Januar',\n",
+                "          'Februar',\n",
+                "          'März',\n",
+                "          'April',\n",
+                "          'Mai',\n",
+                "          'Juni',\n",
+                "          'Juli',\n",
+                "          'August',\n",
+                "          'September',\n",
+                "          'Oktober',\n",
+                "          'November',\n",
+                "          'Dezember']\n",
+                "\n",
+                "plt.plot(np.arange(12), np.random.random(12))\n",
+                "plt.xticks(np.arange(12), months, rotation=45, rotation_mode='anchor', ha='right', va='top')\n",
+                "plt.xlim(0, 11)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "# Histogramme\n",
+                "Sehr häufig braucht man Histogramme."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {
+                "collapsed": true
+            },
+            "outputs": [],
+            "source": [
+                "# Zufallsdaten generieren:\n",
+                "x = np.random.normal(0, 1, 1000)\n",
+                "plt.hist(x, bins=20);\n",
```

Comment:
I understand the `;`, maybe explain at the start and use it with every plot?

Reply:
yes, good idea.